--- linux-60c00a5eccfb875d306bcf39430257df6a535d99/arch/arm64/boot/dts/rockchip/Makefile.orig	2024-02-20 07:00:38.377059633 -0800
+++ linux-60c00a5eccfb875d306bcf39430257df6a535d99/arch/arm64/boot/dts/rockchip/Makefile	2024-02-20 07:01:28.660783297 -0800
@@ -113,6 +113,7 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588-ro
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588-turing-rk1.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-coolpi-4b.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-indiedroid-nova.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-gameforce-ace.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-khadas-edge2.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-rock-5a.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-orangepi-5.dtb
